app/components/Feed/dummydata/index.ts

```diff
@@ -0,0 +1,71 @@
+import { brush, music, pencil, star } from "@/app/assets/icons";
+import { image1, image2 } from "@/app/assets/images";
+import { StaticImageData } from "next/image";
+
+interface Post {
+    creator: string;
+    creatorImage: StaticImageData;
+    art: StaticImageData;
+    artdescription: string;
+    description: string;
+    code: string;
+    licence: string;
+    href: string;
+    id:string;
+  }
+  
+  
+  
+ export const Post:Post[] = [
+    {
```

Comment:
We will be fetching this data from an API and when that arrives you will have to rewrite the code.

Instead, you can add the dummy data to a GitHub gist and load it up from there, so that it is synonymous with pulling from an API endpoint